packages/framework-core/test/booster-graphql-dispatcher.test.ts

```diff
@@ -20,8 +19,7 @@ import { GraphQLWebsocketHandler } from '../src/services/graphql/websocket-proto
 import { ExecutionResult } from 'graphql/execution/execute'
 import { GraphQLError } from 'graphql'
 import { BoosterTokenVerifier } from '../src/booster-token-verifier'
-
-const logger: Logger = console
+import { noopLogger as logger } from './helpers/logger-helper'
```

Comment:
There were tests that leaked logs, I switched their logger to a noop logger

Reply:
Just to consider: Not always is good to silence the logs in the tests. It can help to troubleshoot a failing test, as you can see the logs of the tested code printed out. But I would leave this decision to the maker of each test.

Reply:
I believe that logging should only be enabled when troubleshooting, and then shutdown again